apps/examples/src/examples/screenshot-tool/ScreenshotTool/childStates/Dragging.ts

```diff
@@ -76,17 +76,19 @@ export class ScreenshotDragging extends StateNode {
 				copyAs(
 					editor,
 					shapes.map((s) => s.id),
-					'png',
-					{ bounds: box, background: editor.getInstanceState().exportBackground }
+					{ format: 'png', bounds: box, background: editor.getInstanceState().exportBackground }
```

Comment:
can `editor.getInstanceState().exportBackground` just be the default if not specified?

Reply:
oh yeah this is actually the case now anyway - i can remove this line